### ISSUE TITLE
Teacup Instance Variable Inheritance Feature

### ISSUE THREAD

**Issue by GantMan** [colinta/teacup#77]
Would be great if stylesheets that imported a base stylesheet automatically inherit their parent's instance variables.

e.g. uses sugarcube
File `base.rb`

``` ruby
Teacup::Stylesheet.new :base do
  @back_color = 0xc3a477.uicolor 
end
```

File `settings.rb`

``` ruby
Teacup::Stylesheet.new :settings do
  import :base

  # already has @back_color = 0xc3a477.uicolor 
end
```

**Comment by colinta** [comment]
This will be available in 2.0

https://github.com/colinta/teacup/tree/2.0

**Comment by GantMan** [comment]
Excellent!
